### PR TITLE
Add the error-summary component to the email collection page

### PR DESCRIPTION
**Pull request by tuzz**
https://trello.com/c/HSre7K10/453-style-the-mvp-email-collection-page-needed-by-jan

This pattern is used elsewhere on GOV.UK. It
summarises what went wrong and links to the field
that needs to be fixed.

This element is focussed with JavaScript on page
load which means it’s the first thing screen
readers read out which is a helpful accessibility
pattern.

### CODE REVIEW

File: app/views/subscriptions/new.html.erb [hunk 1]
```diff
@@ -10,12 +10,24 @@
...
-<%= render "components/back_link", {
+<%= render "govuk_publishing_components/components/back_link", {
   href: @back_url,
 } %>
 
 <div class='grid-row'>
   <div class='column-two-thirds'>
+    <% if flash[:error] %>
+      <%= render "govuk_publishing_components/components/error_summary", {
+        title: "This isn't a valid email address",
```
Comment (fofr) [inline app/views/subscriptions/new.html.erb]: Minor: Text could use smart quotes for better readability (http://smartquotesforsmartpeople.com/)

**Comment by fofr** [comment]
Looks good. Is there a screenshot of the new UI?
(#102 would make a thorough review a little easier)

File: app/views/subscriptions/new.html.erb (hunk 1 above)
Comment (rubenarakelyan) [reply app/views/subscriptions/new.html.erb]: Fixed

**Comment by rubenarakelyan** [comment]
screen shot 2018-01-15 at 11 41 45 1

**Pull request merged**